elcid/api.py

```diff
@@ -78,6 +80,26 @@
         LAB_TEST_TAGS[test_name].append(tag)
 
 
+def get_upstream_lab_tests_for_patient(user, patient):
```

Comment:
none of these three methods are tested...

Reply:
resolved on 781046f6